Kickstarter-iOS/Release.entitlements

```diff
@@ -1,34 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-  <dict>
-    <key>aps-environment</key>
-    <string>production</string>
-    <key>com.apple.developer.applesignin</key>
-    <array>
-      <string>Default</string>
-    </array>
-    <key>com.apple.developer.associated-domains</key>
-    <array>
-      <string>applinks:www.kickstarter.com</string>
-      <string>applinks:click.e.kickstarter.com</string>
-      <string>applinks:click.em.kickstarter.com</string>
-      <string>applinks:email.kickstarter.com</string>
-      <string>applinks:emails.kickstarter.com</string>
-      <string>applinks:e2.kickstarter.com</string>
-      <string>applinks:e3.kickstarter.com</string>
-      <string>applinks:clicks.kickstarter.com</string>
-      <string>applinks:ea.kickstarter.com</string>
-      <string>applinks:me.kickstarter.com</string>
-      <string>webcredentials:kickstarter.com</string>
-    </array>
-    <key>com.apple.developer.icloud-container-identifiers</key>
-    <array/>
-    <key>com.apple.developer.in-app-payments</key>
-    <array>
-      <string>merchant.com.kickstarter</string>
-    </array>
-    <key>com.apple.developer.ubiquity-kvstore-identifier</key>
-    <string>$(TeamIdentifierPrefix)$(CFBundleIdentifier)</string>
-  </dict>
+<dict>
+	<key>aps-environment</key>
+	<string>production</string>
+	<key>com.apple.developer.applesignin</key>
+	<array>
+		<string>Default</string>
+	</array>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:www.kickstarter.com</string>
+		<string>applinks:kickstarter.com</string>
+		<string>applinks:emails.kickstarter.com</string>
+		<string>applinks:clicks.kickstarter.com</string>
+		<string>webcredentials:kickstarter.com</string>
```

Comment:
Is this for Apple/Facebook login? Just curious; I noticed it's not in the unit tests or elsewhere in your code changes.

Reply:
The webcredentials? I'm not sure; I think it's a login thing but it has nothing to do with deeplinks (other than them both being entitlements) so I didn't touch it. It just looks like I did, since the spacing changed.